ora2/jobs/RunAcceptanceTests.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    Depends upon the 'Xvfb Plugin'
+
+    * FOLDER_NAME: some-folder
+    * NOTIFY_ON_FAILURE : e-mail-contact@example.com
+    * XVFB_NAME : default-xvfb
+*/
+
+package ora2.jobs
+
+class RunAcceptanceTests {
+
+    public static def job = { dslFactory, extraVars ->
+        return dslFactory.job(extraVars.get('FOLDER_NAME', 'ora2') + '/run-acceptance-tests') {
+            description('Run ORA2 acceptance tests on a sandbox.')
+            logRotator{
+                daysToKeep(10)
+                numToKeep(1)
+                artifactDaysToKeep(1)
+                artifactNumToKeep(1)
+            }
+            properties {
+                githubProjectProperty {
+                    projectUrlStr('https://github.com/edx/edx-ora2')
+                }
+                parameters {
+                    stringParam(
+                        'TEST_HOST',
+                        'ora2.sandbox.edx.org',
+                        "The hostname of the sandbox."
+                    )
+                    stringParam(
+                        'BASIC_AUTH_USER',
+                        'guido',
```

Comment:
Decided to leave these in since the jenkins script in ora depends on them, https://github.com/edx/edx-ora2/blob/master/scripts/jenkins-acceptance-tests.sh it should allow us to at least test against boxes with these credentials in the future.